biosdk-client/README.md

```diff
@@ -1,76 +1,148 @@
-# Bio-SDK client
+# BioSDK-Client
 
-This library provides implementation of [IBioAPI](https://github.com/mosip/commons/blob/master/kernel/kernel-biometrics-api/src/main/java/io/mosip/kernel/biometrics/spi/IBioApi.java) that internally connects with Bio-SDK serivces for Bio SDK related functionality. It can be used by ID authentication & ID repo services to perform 1:N match, segmentation, extraction etc.
+## Overview
+
+The BioSDK-Client library provides an implementation of [IBioAPIV2](https://github.com/mosip/bio-utils/blob/master/kernel-biometrics-api/src/main/java/io/mosip/kernel/biometrics/spi/IBioApiV2.java), enabling seamless integration with Bio-SDK services for biometric-related functionalities. It supports operations like 1:N matching, segmentation, and extraction, making it a critical component for ID authentication and ID repository services.
 
 It is used by:
-* authentication-internal-service
-* authentication-kyc-service
-* authentication-service
-* idrepo-bioextractor-service
 
-## Requirements:
-* Java version = 11.X.X
-* Maven version >= 3.6
+	* authentication-internal-service
+	* authentication-kyc-service
+	* authentication-service
+	* idrepo-bioextractor-service
```

Comment:
where is this project kept ? Or this is the the idrepo-identity service ?

Reply:
idrepo-identity service correct

Reply:
Can we correct this in this PR itself ?